code/game/objects/buckling.dm

```diff
@@ -81,7 +81,7 @@
 	
 	if(do_after(user, buckleTime, src))
 		if(buckle_mob(M))
-			playsound(src.loc, 'sound/machines/Custom_closetopen.ogg', 15, 1, -3)
+			playsound(src.loc, 'sound/machines/Custom_closetopen.ogg', 65, 1, -3)
```

Comment:
does this need to be so loud? what's the reasoning behind this

loud sounds are rather intrusive,

Reply:
they are not, they barely noticable, before they couldnt be heard at all